src/fs/rock/RockRebuild.h

```diff
@@ -54,7 +54,7 @@ class Rebuild: public AsyncJob, private IndependentRunner
 
 protected:
     /// whether the current kid is responsible for rebuilding this db file
```

Comment:
The best I can suggest is a combination of two changes:

```suggestion
    /// whether the current kid is responsible for rebuilding the given cache_dir
```

And then revert the parameter-removal change. Just do not _name_ the (unused) parameter instead.

Rationale: The parameter is unused by the current IsResponsible() implementation because it is not needed to provide the correct answer given how we decide who is responsible for cache_dir rebuilding today. However, from the caller point of view, it is a lot more clearer to supply the cache_dir instead of relying on the subtle invariants that make the cache_dir value irrelevant. Those invariants are also difficult to document in a short method description.

It may be possible to refactor the callers to clarify all this without relying on unused parameters, but such a rewrite would not be simple and would be out of this PR scope.

Reply:
Done in commit 0846a584c to be pushed soon